pkg/feature/manifest.go

```diff
@@ -12,11 +12,17 @@ import (
 
 	"github.com/ghodss/yaml"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/metadata/annotations"
 )
 
+const annotationTrue = "true"
+
 type Manifest interface {
 	// Process allows any arbitrary struct to be passed and used while processing the content of the manifest.
 	Process(data any) ([]*unstructured.Unstructured, error)
+	// SetManaged sets all non-patch objects to be managed/reconciled by setting the annotation.
+	SetManaged([]*unstructured.Unstructured) []*unstructured.Unstructured
```

Comment:
I am not sure about the `Set`... being Java old-timer :) `MarkAsManaged` maybe?

Also if you pass a pointer to the slice there is no need to return it back, right?

TIL you can have unnamed input params in the interface btw. Would never thought about needing it. Though... maybe add `objects` here :)

Reply:
[623a7ca](https://github.com/opendatahub-io/opendatahub-operator/pull/974/commits/623a7cac73006015defc922d5645a59bf52f8e17) done here